editor/test/integration/engine/native_extensions_test.clj

```diff
@@ -43,12 +43,12 @@
                (set (map resource/proj-path (native-extensions/engine-extension-roots project evaluation-context)))))))))
 
 (deftest unpack-bin-zip-test
-  (testing "${ext}/plugins/bin/${platform}.zip is extracted to /build/plugins/${ext}/plugins/bin/platform/ folder"
+  (testing "${ext}/plugins/${platform}.zip is extracted to /build/plugins/${ext}/plugins/ folder"
    (with-clean-system
      (let [workspace (test-util/setup-workspace! world "test/resources/extension_project")
            _ (test-util/setup-project! workspace)
            root (g/node-value workspace :root)]
-       (is (.exists (io/file (str root "/ext_with_bin_zip/plugins/bin/x86_64-macos.zip"))))
+       (is (.exists (io/file (str root "/ext_with_bin_zip/plugins/x86_64-macos.zip"))))
        (is (not (.exists (io/file (str root "/ext_with_bin_zip/plugins/bin/x86_64-macos/lsp.editor_script")))))
```

Comment:
Should this next path include `/bin/` now that you've stripped it from the line above?

In fact, this test looks suspect to me. It seems the `testing` description does not match what we check for? Are you sure this is correct? If so, I think it needs to be clarified somewhat with some comments.

Reply:
No, the unchanged path is correct. Previously, we had `plugins/bin/x86_64-macos.zip` that contained a single file at the root (`/lsp.editor_script`). Now we have `plugins/x86_64-macos.zip` that contains a file at a different path (`/bin/x86_64-macos/lsp.editor_script`), so the result should be the same, hence the only change in test is the contents and location of the zip file.

Reply:
I think adding a comment detailing the file structure of the zip file would help.

Reply:
Good point, I'll add it!